opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -76,3 +76,16 @@ def on_end(self, span: Span) -> None:
 
     def shutdown(self) -> None:
         self.span_exporter.shutdown()
+
+
+class ConsoleSpanExporter(SpanExporter):
+    """Implementation of :class:`Exporter` that prints spans to the console.
+
+    This class can be used for diagnostic purposes. It prints the exported
+    spans to the console STDOUT.
+    """
+
+    def export(self, spans: typing.Sequence[Span]) -> SpanExportResult:
+        for span in spans:
+            print(span)
+        return SpanExportResult.SUCCESS
```

Comment:
We should implement `shutdown()` method as well.

Reply:
This is a sync exporter which inherited the no-op `shutdown()` from its parent.